sixtyfps_compiler/generator/cpp.rs

```diff
@@ -2128,7 +2128,7 @@ fn compile_path(path: &crate::expression_tree::Path, component: &Rc<Component>)
     }
 }
 
-fn compile_path_events(events: &crate::expression_tree::PathEvents) -> (Vec<String>, Vec<String>) {
+fn compile_path_events(events: &[crate::expression_tree::PathEvent]) -> (Vec<String>, Vec<String>) {
```

Comment:
How is that better?

Reply:
Ah yes, because it was a Vec before

Reply:
This accepts anything that can be turned into a slice, not just a Vec.

It also needs less derefs to get to the actual data, so it might even be a little bit faster.